journal-week-1.md

```diff
@@ -0,0 +1,16 @@
+# One Down Looking Forward for Eleven More 
+
+
+The first week was a shock, I indeed entered a boot camp. No hanky panky. Starting from the prep-work and the rest of the week, all of us from the class had the taste of brain squeezing experience. 
```

Comment:
@kellymurray, uh oh, i just goggled it and saw the meaning, we have different meaning of that in our hometown. will definitely delete it though.